app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ProfilesController < ApplicationController
+   def show
+   end
+
+   def update
+      redirect_to dashboard_path if current_user.update profile_params
```

Comment:
yeah - I am still working through the proper way to do these redirects, particularly keeping security in mind (i.e. we don't want to expose /users or editing of any individual user to just any member who is logged in)